Add Afrikaans Bible

rudolfbyker opened the pull request:
Greetings from South Africa! I started adding an Afrikaans bible text, since I would like to use your python module in one of my projects. (I could also help with other bugfixes, PEP8 compliance, etc.)

rudolfbyker committed:
Add Afrikaans Bible

davisd commented:
Greetings!  I would love to add this to the texts.  Are you happy with your pull request?  You say that you started adding it- did you complete all of the books to your satisfaction?

Let me know if you're happy with it and I'll put another set of eyes on it.  if it checks out, I'll merge.  Thank you for your contribution!

-David

rudolfbyker commented:
Yes and no... I added all of the books, but there are a few differences in the versification of the Afrikaans and English translations, notably in the Psalms, where the heading is verse 1, and the English verse 1 is verse 2 in Afrikaans, etc. I don't have time to check everything, so I just used the number of verses from the English module, and added a TODO message about that.

There are two other TODOs, which are questions for you:

1. Is it OK to use non-ASCII characters in the dictionary keys? The book of Joel in Afrikaans is spelled "Joël", which means that the abbreviation also contains the "ë".
2. Many Afrikaans books start with "Die" (the) or have even longer titles, like "Aan die Romeine" (To the Romans) instead of "Romeine" (Romans). Which ones should we use?
